keyboards/kodeboard/info.json

```
@@ -0,0 +1,62 @@
+{
+    "manufacturer": "MUKIL",
+    "keyboard_name": "k0depad",
+    "maintainer": "bukil",
+    "bootloader": "rp2040",
+    "processor": "RP2040",
+    "diode_direction": "ROW2COL",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true
+    },
+    "dynamic_keymap": {
+        "layer_count": 8
+    }
+    "encoder": {
+        "enabled": true,
+        "rotary": [
+            {"pin_a": "GP0", "pin_b": "GP1"}
+        ]
+    },
+    "matrix_pins": {
+        "cols": ["GP16", "GP17", "GP18", "GP19"],
+        "rows": ["GP15", "GP14", "GP13", "GP12"]
+    },
+    
+    "url": "www.github.com/bukil",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x0001",
+        "vid": "0x4D4B"
+    },
+    "community_layouts": ["k0depad"],
```

This isn't actually a recognized layout.  However, it does look to be an ortho 4x4 type macropad. 
```suggestion
    "community_layouts": ["ortho_4x4"],
```